src/collection/style.js

```diff
@@ -71,6 +71,25 @@ var elesfn = ({
     }
   },
 
+  // get the associated values at the specified index when parsing a property with a multiple tag
+  parsedStyleMult: function( property, index ){
+    var ele = this[0];
```

Comment:
Ok, so at the moment, if the index is not found, it defaults to the default settings in getDefaultProperty. So it is ok to default to index 0 instead?